lib/services/workflow-api-service.js

```diff
@@ -155,7 +237,32 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.find({ node: target })
+            .then(function(workflows){
+                return _.filter(workflows, function(wf) {
+                    return wf.active();
```

Comment:
@dalebremner Correct me if I'm wrong, but I think the idea was to make this a bit more behavioral.  The code reads as "filter for active workflows" rather than "query for _status equals pending".  
The reason for not using the task graph store is that `taskGraphStore.findActiveGraphForTarget` goes directly to Mongo which results in returned resources that are inconsistent with the /workflows and /workflows/:id